samples/matter/light_bulb/README.rst

```diff
@@ -119,6 +122,68 @@ Factory data support
+
+The sample can be configured to communicate with `AWS IoT Core`_ to control attributes in supported clusters on the device.
+After the setup, the sample will mirror these attributes in the AWS IoT shadow document.
+This makes it possible to remotely control the device using the `AWS IoT Device Shadow Service`_.
+The supported attributes are ``onoff`` and ``levelcontrol``.
+
+The following figure illustrates the relationship between the AWS IoT integration layer and the light bulb sample:
+
+.. figure:: /images/aws_matter_integration.svg
+   :alt: Sample implementation of the AWS IoT integration layer
+
+   AWS IoT integration layer implementation diagram
+
+The following figure illustrates the interaction with the AWS IoT Shadow Service:
+
+
+.. figure:: images/aws_matter_interaction.svg
+   :alt: Interaction with the AWS IoT Shadow Service
+
+   AWS IoT Shadow Servic
+
+AWS IoT setup and configuration
+-------------------------------
+
+To set up an AWS IoT instance and configure the sample, complete the following steps:
+
+1. Complete the setup and configuration described in the :ref:`lib_aws_iot` documentation to get the host name, device ID, and certificates used in the connection.
+#. Set the :kconfig:option:`CONFIG_AWS_IOT_BROKER_HOST_NAME` and :kconfig:option:`CONFIG_AWS_IOT_CLIENT_ID_STATIC` options in the :file:`overlay-aws-iot-integration.conf` file.
+#. Import the certificates mentioned in the :file:`light_bulb/src/aws_iot_integration/certs` file.
+#. Build the sample using the following command:
+
+   .. code-block:: console
+
+      west build -p -b nrf7002dk_nrf5340_cpuapp -- -DOVERLAY_CONFIG="overlay-aws-iot-integration.conf"
+
+#. Flash the firmware, boot the sample, and commission the device to the Matter network.
+   See `Commissioning the device`_ for more information.
+#. |connect_kit|
+#. |connect_terminal_ANSI|
+#. Observe that the device automatically connects to AWS IoT when an IP is obtained and is able to maintain the connection.
+#. Use the following bash function to populate the desired section of the shadow:
+
+   .. code-block:: console
+
+      function aws-update-desired() {
+            aws iot-data update-thing-shadow --cli-binary-format raw-in-base64-out --thing-name my-thing --payload "{\"state\":{\"desired\":{\"onoff\":$1,\"level_control\":$2}}}" "output.txt"
+      }
+
+   You can also use ``aws-update-desired 0 0``, or ``aws-update-desired 1 128 (onoff, levelcontrol)``.
+   Alternatively, you can alter the device shadow directly through the `AWS IoT console`_.
+
+#. Observe that the light bulb changes state.
+   The local changes to the attributes always take precedence over what is set in the shadow's desired state.
+
+.. note::
+   The integration layer has built-in reconnection logic and tries to maintain the connection as long as the device is connected to the internet.
+   The reconnection interval can be configured using the ``CONFIG_AWS_IOT_RECONNECTION_INTERVAL_SECONDS`` option.
```

Comment:
Should this be a Kconfig option?

Reply:
Yeah, this is a Kconfig option. But this is not defined in the Kconfig library, so don't link. This is the reason to keep it as backticks.

Reply:
Could still have it as "the [...] Kconfig option", but it's not a blocker for me.